mods/cnc/tilesets/winter.yaml

```diff
@@ -589,8 +589,8 @@ Templates:
 		Size: 2,2
 		Category: Cliffs
 		Tiles:
-			0: Clear
-			1: Clear
+			0: Rock
```

Comment:
And here:
![tdcliff6](https://cloud.githubusercontent.com/assets/7704140/12011143/3b8239ca-acc3-11e5-9f8a-db6ac4a98e05.png)